index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+    <header class="heading">
+        <h1 id="title">freeCodeCamp Survey <Form></Form>
+        </h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </header>
+    <form action="" id="survey-form">
+        <div class="content">
+            <label id="name-label" for="name">Name</label>
+        <input type="text" id="name" placeholder="Enter your name" required>
+        </div>
+        
+        <div class="content">
+            <label id="email-label" for="email">Email</label>
+            <input type="email" id="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your email" required>
+            
+
+        </div>
+       
+        <div class="content">
```

Comment:
- [ ]  Remove unnecessary spaces to make the code more readable.
- [ ] I would suggest that you use fewer "id" selectors and more "class" selectors.